app/lib/features/home/providers/navigation.dart

```diff
@@ -12,7 +12,7 @@ import 'package:acter/common/themes/app_theme.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 
 const fallbackSidebarIdx = 1;
-const fallbackBottomBarIdx = 0;
+const fallbackBottomBarIdx = 1;
```

Comment:
I am not sure this is correct. The fallback is used _every time_ we are not matching correctly - so all `/tasks` `/pins`-pages as well. I think if you used the quick-jump and selected an item (search -> one of the icons) you'll find the wrong icon highlighted on the bottom now: not the home but the news is highlighted ....

That's not the behavior we want.

Reply:
There is conditional routes for Mobile and Desktop.
 
![Screenshot 2023-10-26 at 5 03 34 PM](https://github.com/acterglobal/a3/assets/51526480/d03fbbb6-1d89-4824-b569-d4cd876ca6cf)

On Desktop: Home Tab is considered as default tab and route
On Mobile: Update Tab is considered as default tab but when we do login at that time index was returning as -1 and that's why home bottom was selected even though Update page is opened

Please share your thoughts here.

Reply:
Yeah, but the code is more complex. As described in the comment: the fallback is used for more than just this one case. If you follow the path described above: does it show the wrong icon? Then the patch isn't sufficient.

Reply:
I have removed this point from this PR. I will handle it on subsequent PR.